### PR TITLE
Create Gateway U2fVerificationBundle with database connection and migrations

### PR DESCRIPTION
**Pull request by rjkip**

### CODE REVIEW

**Comment by rjkip** [comment]
@DRvanR 

A wild bundle with U2F database connection and migrations support appears!

File: src/Surfnet/StepupGateway/GatewayBundle/Console/Command/MigrationsDiffDoctrineCommand.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+use Symfony\Component\Process\ProcessBuilder;
+
+/**
+ * @SuppressWarnings(PHPMD.UnusedLocalVariables)
+ */
+class MigrationsDiffDoctrineCommand extends Command
+{
+    protected function configure()
+    {
+        $this->setName('gateway:migrations:diff');
```
Comment (DRvanR) [inline src/Surfnet/StepupGateway/GatewayBundle/Console/Command/MigrationsDiffDoctrineCommand.php]: think I'd rename it `u2f:migrations:diff` (same for the others), as it has nothing to do with the gateway...

**Comment by DRvanR** [comment]
bar the naming :+1:

**Comment by rjkip** [comment]
:8ball: @DRvanR

File: src/Surfnet/StepupGateway/U2fVerificationBundle/Console/Command/MigrationsDiffDoctrineCommand.php [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    protected function configure()
+    {
+        $this->setName('u2f:migrations:diff');
+        $this->setDescription('Performs a mapping/database diff using the correct entity manager');
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        $output->writeln(['<info>Generating diff for U2F...</info>', '']);
```
Comment (DRvanR) [inline src/Surfnet/StepupGateway/U2fVerificationBundle/Console/Command/MigrationsDiffDoctrineCommand.php]: an aside: I've grown accustomed to using `$output->write('<info>Generating diff for U2F...</info>', true)`. Dunno, to me seems more logical that "write line[lines]" :wink:

**Comment by DRvanR** [comment]
nice! :+1:

File: src/Surfnet/StepupGateway/U2fVerificationBundle/Console/Command/MigrationsDiffDoctrineCommand.php (hunk 2 above)
Comment (rjkip) [inline src/Surfnet/StepupGateway/U2fVerificationBundle/Console/Command/MigrationsDiffDoctrineCommand.php]: Next time, Gadget.

**Pull request merged**